generators/entity/index.js

```diff
@@ -1664,7 +1760,8 @@ module.exports = EntityGenerator.extend({
         },
 
         writeClientTestFiles: function () {
-            if (this.skipClient) return;
+            if(this.skipClient) return;
```

Comment:
this `if (this.skipClient) return;` is correct formatting with space between things, I guess you have removed the space in many places can you plz check your diff